database/rrdset.c

```diff
@@ -932,15 +936,6 @@ RRDSET *rrdset_create_custom(
 
     store_active_chart(st->chart_uuid);
 
-#ifdef ENABLE_ACLK
-    host->obsolete_count = 0;
-#endif
-    rrdhost_cleanup_obsolete_charts(host);
-#ifdef ENABLE_ACLK
-    if (host->obsolete_count)
-        aclk_update_chart(st->rrdhost, "dummy-chart", ACLK_CMD_CHARTDEL);
```

Comment:
@underhood: Under which situation can we have a dummy chart after the cleanup process?

Reply:
> @underhood: Under which situation can we have a dummy chart after the cleanup process?

The ACLK_CMD_CHARTDEL will not send a delete for a specific chart so in that sense it is dummy (not used at all).